config.go

```diff
@@ -70,16 +72,61 @@ func (cfg Config) MakeLabels(types []string) string {
 }
 
 // GetNativePropertyValue is called during building properties for save and when the expanded property key exists in the config.
-func (cfg Config) GetNativePropertyValue(node *lpg.Node, expandedPropertyKey, val string) interface{} {
-	if _, exists := cfg.PropertyTypes[expandedPropertyKey]; exists {
-		va := ls.GetValueAccessor(cfg.PropertyTypes[expandedPropertyKey])
-		native, err := va.GetNativeValue(val, node)
+func (cfg Config) GetNeo4jPropertyValue(expandedPropertyKey string, val string) (interface{}, error) {
+	prop, exists := cfg.PropertyTypes[expandedPropertyKey]
+	propType := strings.Split(prop, ",")
```

Comment:
Move exists check before split
Split won't work if the date/time pattern has comma in it.
Use indexOf ","